File: JavaScript_Courses/JavaScript_Basics/Preparation/04_For_Loop_Exercise/05_Salary.js
function salary(input) {
    let penaltyFacebook = 150;
    let penaltyInstagram = 100;
    let penaltyReddit = 50;


    let openedTabs = Number(input[0]);
    let salary = Number(input[1]);

    // for (let i = 2; i < input.length; i++) {
    for (let i = 0; i < openedTabs; i++) {
        let website = input[i + 2];
        // console.log(website);
        if (website == "Facebook") {
            salary -= penaltyFacebook;
        } else if (website == "Instagram") {
            salary -= penaltyInstagram;
        } else if (website == "Reddit") {
            salary -= penaltyReddit;
        }
    }

    if (salary <= 0) {
        console.log("You have lost your salary.");
    } else {
        console.log(salary);
    }

}

// salary(["10", "750", "Facebook", "Dev.bg", "Instagram", "Facebook", "Reddit", "Facebook", "Facebook"]);

File: Python_Courses/Python_Basics/Python_PB_2025_03/04_For_Loop_Exercise/05_Salary.py
opened_tabs = int(input())
salary = float(input())

for _ in range(opened_tabs):  # [0, 1,... opened_tabs) == [0, ... opened_tabs - 1]
    current_tab = input()

    if current_tab == 'Facebook':
        salary -= 150
        if salary <= 0:
            break
    elif current_tab == 'Instagram':
        salary -= 100
        if salary <= 0:
            break
    elif current_tab == 'Reddit':
        salary -= 50
        if salary <= 0:
            break


if salary <= 0:
    print('You have lost your salary.')
else:
    print(int(salary))
